src/ServiceControl/Contracts/Operations/FailureDetails.cs

```diff
@@ -4,6 +4,11 @@ namespace ServiceControl.Contracts.Operations
 
     public class FailureDetails
     {
+        public FailureDetails()
+        {
+            TimeOfFailure = DateTime.UtcNow;
```

Comment:
Is this correct. Is there ever a case where this does not get set? I thought we default this if we don't get a header.